ETC.tsv

```diff
@@ -11802,7 +11802,7 @@ ETC_20150714_011803	You've found Moas!{nl}Defeat Moas and collect the lumps that
 ETC_20150714_011804	Thorny Vines are spreading evil energies!
 ETC_20150714_011805	{memo X}Protect the crystal of the authority
 ETC_20150714_011806	The portal suddenly opened!
-ETC_20150714_011807	Monsters started to attack the sealed tower!
+ETC_20150714_011807	Monsters started to attack the seal tower!
```

Comment:
Im not entirely sure on the context here, but would like a clarification that seal tower is something that seals, and not a tower that is sealed.